.github/workflows/acceptance.yml

```diff
@@ -89,10 +89,15 @@ jobs:
 
       - uses: volta-cli/action@v1
 
+      - name: Set python version output
+        id: python-version
+        run: |
+          echo "::set-output name=python-version::$(grep "3.6" .python-version)"
```

Comment:
Do we need to grep `3.6` here? Wouldn't it be better to use whatever is in `.python-version`, or at least the first line?

Reply:
@armenzg Let's just use the first line throughout everything now.